webapp-django/package.json

```diff
@@ -8,10 +8,28 @@
   },
   "author": "Mozilla",
   "license": "MPL-2.0",
-  "dependencies": {
+  "devDependencies": {
     "cssmin": "0.4.3",
     "graceful-fs": "4.1.4",
     "less": "2.7.1",
     "uglify-js": "2.6.1"
+  },
+  "dependencies": {
+    "Select2": "github:select2/select2#3.5.4",
+    "ace-builds": "1.2.8",
+    "d3": "3.5.17",
+    "d3-sankey": "0.7.1",
+    "filesize": "2.0.3",
+    "flatpickr": "4.0.3",
+    "font-awesome": "4.6.1",
+    "jquery": "2.1.0",
+    "jquery-jsonview": "1.2.3",
+    "jquery-ui": "github:jquery/jquery-ui#1.11.1",
```

Comment:
jQuery didn't start uploading to npm until 1.12, I believe. We'll update this in a future PR.